### ISSUE TITLE
Add support for visualizing reactive streams in a remote app

### ISSUE THREAD

**Issue by whitelynx** [jaredly/rxvision#6]
This would probably involve splitting rxvision into an "interceptor" module and a "visualizer" module, and adding a user-settable "transport" function/object to the visualizer.

Then, the user can implement whatever transport they need between the two, allowing you to visualize events in e.g. a Node.js app. The transport could default to just directly consuming from a local interceptor, in order to support the current use case.

**Comment by jaredly** [comment]
yeah, that sounds like a cool idea. As the code currently stands, that's totally possible.

Include `rxvision.js`, and you get the "tracer".
If you also include `viz.js`, then you also get the visualization too.

You can see [here](https://github.com/jaredly/rxvision/blob/master/run/viz.js) how the viz tool listens to the tracer. It would be simple to add a listener that shoves events onto a websocket or something
